create-k8s-chained-sessions.sh

```diff
@@ -20,37 +19,53 @@ declare REGION='us-east-1'
 function createLeappSession {
     parent_session_name=$1
     chained_session_name="chained-from-${parent_session_name}"
-    echo "starting session for ${parent_session_name} to get role arn"
-    # this has funky piping because `--filter` is a fuzzy lookup and `panorama-k8s-playground` fuzzy matches `panorama-k8s-playground-2` as the first result
```

Comment:
I learned in `school-supplies` that you can use regex anchors (e.g. `^` and `$`) in `--filter`, so I moved this to a helper function and used that instead of `sort | sed | awk`.